hr_timesheet_hours_billed/models/sale_order_line.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2023-nowdays Cetmix OU (https://cetmix.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import api, models
+from odoo.osv import expression
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    @api.depends(
+        "qty_delivered_method",
+        "analytic_line_ids.so_line",
+        "analytic_line_ids.unit_amount_billed",
+        "analytic_line_ids.product_uom_id",
+        "analytic_line_ids.approved",
+    )
+    def _compute_qty_delivered(self):
+        """This method compute the delivered quantity of the SO lines:
```

Comment:
This sounds more like a comment. A relevant info would be that you added  some more field to `depends`

Reply:
I need to write after this text something like
"Added analytic_line_ids.so_line, analytic_line_ids.unit_amount_billed, ..."?

Reply:
@simahawk?